docs/samples/Microsoft.ML.Samples/Dynamic/IidSpikeDetectorTransform.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.ML.Samples.Dynamic
 {
-    public partial class TransformSamples
+    public static partial class TransformSamples
```

Comment:
>TransformSamples [](start = 32, length = 16)

All timeseries example slightly different from our current samples.
They share same class and introduce data classes inside it, instead of using sample utils.
But I'm not sure they transformation to how we see samples should be done in your PR.